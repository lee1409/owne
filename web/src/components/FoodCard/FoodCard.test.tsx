import { render } from "@testing-library/react"
import { FoodCard } from "."

test("FoodCard", () => {
  const foodCard = render(<FoodCard src="#" name="Test" />)
  expect(foodCard).toMatchSnapshot("FoodCard")
})
