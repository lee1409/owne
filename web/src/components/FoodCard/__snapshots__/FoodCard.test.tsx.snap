// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodCard: FoodCard 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="div-root-1 div-root-2"
      >
        <div
          class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            class="MuiCardMedia-root WithStyles(ForwardRef(CardMedia))-root-3 WithStyles(ForwardRef(CardMedia))-root-4"
            style="background-image: url(#);"
            title="ChickenSoup"
          />
        </div>
        <p
          class="MuiTypography-root WithStyles(ForwardRef(Typography))-root-7 Styled(WithStyles(ForwardRef(Typography)))-root-5 Styled(WithStyles(ForwardRef(Typography)))-root-6 MuiTypography-body1"
        >
          Test
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="div-root-1 div-root-2"
    >
      <div
        class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="MuiCardMedia-root WithStyles(ForwardRef(CardMedia))-root-3 WithStyles(ForwardRef(CardMedia))-root-4"
          style="background-image: url(#);"
          title="ChickenSoup"
        />
      </div>
      <p
        class="MuiTypography-root WithStyles(ForwardRef(Typography))-root-7 Styled(WithStyles(ForwardRef(Typography)))-root-5 Styled(WithStyles(ForwardRef(Typography)))-root-6 MuiTypography-body1"
      >
        Test
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
